test/test_game.py

```diff
@@ -31,7 +31,7 @@ def test_full_board_game_over(self, mock_print):
         game = Game()
         total_marks_on_board = 9
         game.get_prompt(total_marks_on_board)
-        mock_print.assert_called_with("Game Over!")
+        mock_print.assert_called_with("Game over - it's a draw!")
```

Comment:
A next step would be to update these game tests to stop testing messages.  Messages should be decoupled from the game.  I think it would be ok to test based on rules though, because the game is dependent on rules.

Reply:
I'm not sure I'm clear on the differentiation or how I would do that...can you elaborate?

For this, my intention is to test the "rule" that if there are 9 marks on the board, the game will tell the player "game over".

Reply:
Let me know if you want to pair to review this comment, so you can get a feel for the feedback I am trying to share.